src/entities/addLiquidity/addLiquidityV2/index.ts

```diff
@@ -34,7 +34,11 @@ export class AddLiquidityV2 implements AddLiquidityBase {
 
     public getAddLiquidity(poolType: string): AddLiquidityBase {
         if (!this.addLiquidityTypes[poolType]) {
-            throw new Error(`Unsupported pool type ${poolType}`);
+            throw new SDKError(
+                'Input Validation',
+                'Add Liquidity',
+                `Unsupported pool type ${poolType}`,
+            );
```

Comment:
What is the initial reason to throw via `SDKERROR` instead of an imported one from utils?

Reply:
It's just a draft so you can provide early feedback, so it's not consistent yet.
I plan to have any error that could/should be reused to be exported to errors.ts file and sourced from there.
And I think that errors that won't be reused could be instantiated like this, but I don't mind if we decide that all errors should be on errors.ts disregard of it being reused or not.